### PR TITLE
Added recipes for flagged triangular lattices and avg. shortest path length

### PR DESCRIPTION
**Pull request by thomasrobiglio**
related to #504

### CODE REVIEW

File: docs/source/api/recipes/recipes.ipynb [hunk 1]
```diff
@@ -85,7 +85,7 @@
    "outputs": [
```
Comment (nwlandry) [inline docs/source/api/recipes/recipes.ipynb:85]: <p>"length" not "lenght", "an" not "and", "remove length-0 paths" not "remove lengths 0". The last sentence is a bit confusing to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (nwlandry) [inline docs/source/api/recipes/recipes.ipynb:85]: <pre class="ql-syntax" spellcheck="false">Line #17.    avg_shortest_path = np.sum(lens) / (N * (N-1))
</pre><p>Where is N defined?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (nwlandry) [reply docs/source/api/recipes/recipes.ipynb:85]: I feel like the recipe should be self-contained.

**Comment by nwlandry** [comment]
Overall, looks good @thomasrobiglio! Nice work! Just a few minor comments to fix and I will approve so that you can merge.

**Review by nwlandry — APPROVED** [review]

**Comment by thomasrobiglio** [comment]
Thank you @nwlandry, I might have been a bit tired when I did the commits :)

**Pull request merged**